lib/commons/text/native-text-alternative.js

```diff
@@ -1,6 +1,5 @@
 import getRole from '../aria/get-role';
-import matches from '../matches/matches';
-import nativeElementType from './native-element-type';
+import getElementSpec from '../standards/get-element-spec';
```

Comment:
Changes in this file should go into a separate PR.

Reply:
Need the property change for virtual node so i think changing it to use `elmSpec` is fine